test/images/Makefile

```diff
@@ -14,7 +14,7 @@
 
 REGISTRY ?= gcr.io/kubernetes-e2e-test-images
 GOARM=7
-QEMUVERSION=v2.7.0
+QEMUVERSION=v2.9.1
```

Comment:
Not really sure whether we need to push all the images under `test/images` with this new qemu binary. @luxas WDYT?